requirements.txt

```diff
@@ -0,0 +1,9 @@
+numpy~=1.15.0
+periodictable~=1.5.3
+picmistandard==0.0.13
+scipy~=1.6.0  # picmistandard bug: https://github.com/picmi-standard/picmi/pull/34
```

Comment:
PR open in: https://github.com/picmi-standard/picmi/pull/34

Reply:
does not hurt to keep this here until the next release of `picmistandard` is out